tools/generator/templates/player/python/interface.cc.jinja2

```diff
@@ -311,6 +311,30 @@ static void _init_python()
     py_module = _import_module("api");
 }
 
+// Flush stdout and stderr
+static void flush_std_files()
+{
+    PyObject *fout = PySys_GetObject("stdout");
+    PyObject *ferr = PySys_GetObject("stderr");
+    PyObject *tmp;
+
+    if (fout != nullptr && fout != Py_None) {
+        tmp = PyObject_CallMethod(fout, "flush", "");
+        if (tmp == nullptr)
+            PyErr_WriteUnraisable(fout);
+        else
+            Py_DECREF(tmp);
+    }
+
+    if (ferr != nullptr && ferr != Py_None) {
+        tmp = PyObject_CallMethod(ferr, "flush", "");
+        if (tmp == nullptr)
+            PyErr_Clear();
```

Comment:
Why the different error handing here? Because the error would go to stderr so it makes no sense to try to write to stderr if we couldn't flush?

Reply:
idk, this whole function is literally copy pasted from CPython.